dataline-server/src/main/java/io/dataline/server/apis/ConfigurationApi.java

```diff
@@ -143,45 +149,53 @@ public SourceRead getSource(@Valid SourceIdRequestBody sourceIdRequestBody) {
   // SOURCE SPECIFICATION
 
   @Override
-  public SourceSpecificationRead getSourceSpecification(@Valid SourceIdRequestBody sourceIdRequestBody) {
+  public SourceSpecificationRead getSourceSpecification(
```

Comment:
Why is it behaving like that for you? Can you adjust your settings?

Reply:
i think it is fixed now.